keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -222,16 +222,16 @@ the queueManager holds the evicted workloads as inadmissible workloads while exp
 Duration this time, other workloads will have a chance to be admitted.
 
 The queueManager calculates an exponential backoff duration by [the Step function](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait@v0.29.1#Backoff.Step)
-according to the $1.41284738^{(n-1)}+Rand$ where the $n$ represents the `workloadStatus.requeueState.count`, and the $Rand$ represents the random jitter.
+according to the $10s*2^{(n-1)}+Rand$ where the $n$ represents the `workloadStatus.requeueState.count`, and the $Rand$ represents the random jitter.
 
-Considering the `.waitForPodsReady.timeout` (default: 300 seconds),
-this duration indicates that an evicted workload with `PodsReadyTimeout` reason is continued re-queuing 
-for the following period where the $t$ represents `.waitForPodsReady.timeout`:
+It will spend awaiting to be requeued after eviction:
+$$\sum_{k=1}^{n}(10s*2^{(k-1)} + Rand)$$
 
-$$t(n+1) + \sum_{k=1}^{n}(1.41284738^{(k-1)} + Rand)$$
+Assuming the workload is requeued 10 times after failing to have all pods ready,
+then the total time awaiting for requeue will take (neglecting the jitter): `10s+20s+40s +...+7680s=2h 8min`.
 
-Given that the `backoffLimitCount` equals `30` and the `waitForPodsReady.timeout` equals `300` (default),
-the result equals 24 hours (+ $Rand$ seconds).
+For comparison, considering `.waitForPodsReady.timeout=300s` (default),
+the workload will spend `50min` total waiting for pods ready.
```

Comment:
Done, but I set it to 10 now PTAL

Reply:
I meant the `backoffLimitCount` in Config API. In this PR, we introduced the fixed value to `Duration` in backoff calculation, right?

Reply:
`// When it is null, the workloads will repeatedly and endless re-queueing.` Isn't that enough?

Reply:
> I meant the backoffLimitCount in Config API. In this PR, we introduced the fixed value to Duration in backoff calculation, right?

I discussed this with @mimowo offline. It seemed that our understanding was missing each other, but we could sync our opinions.

> // When it is null, the workloads will repeatedly and endless re-queueing. Isn't that enough?

I'm ok with removing this sentence since it seems that this example seems to lose importance.